src/spicy/manager.cc

```diff
@@ -699,14 +702,44 @@ void Manager::InitPostScript() {
         if ( ! tag )
             reporter->InternalError("cannot get analyzer tag for '%s'", p.name_analyzer.c_str());
 
+        auto register_analyzer_for_port = [&](auto tag, const hilti::rt::Port& port_) {
+            SPICY_DEBUG(hilti::rt::fmt("  Scheduling analyzer for port %s", port_));
+
+            // Well-known ports are registered in scriptland, so we'll raise an
+            // event that will do it for us through a predefined handler.
+            zeek::Args vals = Args();
+            vals.emplace_back(tag.AsVal());
+            vals.emplace_back(zeek::spicy::rt::to_val(port_, base_type(TYPE_PORT)));
+            EventHandlerPtr handler = event_registry->Register("spicy_analyzer_for_port");
+            event_mgr.Enqueue(handler, vals);
```

Comment:
Another nit, there's a vaargs function, that voids needing the explicit args (and I wonder if the lambda could go then).
```
event_mgr.Enqueue(handler, tag.AsVal(), zeek::spicy::rt::to_val(port_, base_type(TYPE_PORT)));
```

Ah, but that pattern is used for mimes, too. Never mind.

Reply:
Used elsewhere in the Spicy code too iirc, so I'll leave it for now, could clean up/modernize some other time.